src/Services/Importers/Chunk.php

```diff
@@ -69,13 +72,21 @@ public function run(): void
         }
     }
 
-    private function auth(): void
+    private function authenticate(): void
     {
         if ($this->importer instanceof Authenticates) {
             Auth::setUser($this->user);
         }
     }
 
+    private function authorize(): void
+    {
+        if ($this->importer instanceof Authorizes
+            && $this->importer->authorize($this->user, $this->params) === false) {
```

Comment:
why not use `!` here?

Reply:
because I want to use laravel convention here
only `return false` is assumed as `unauthorized`